mantis-server/mantis-server-agent/src/main/java/io/mantisrx/server/agent/BlobStore.java

```diff
@@ -67,6 +72,23 @@ static BlobStore forHadoopFileSystem(URI clusterStoragePath, File localStoreDir)
                 .withThreadSafeBlobStore();
     }
 
+    static BlobStore forHadoopFileSystem(URI clusterStoragePath, URI fallbackStoragePath, File localStoreDir)
```

Comment:
nit: why custom API for this?

Reply:
what do you mean by custom API? This is overloading the same API above with extra param.